judo-tatami-asm2sdk/src/main/java/hu/blackbelt/judo/tatami/asm2sdk/Asm2SDKWork.java

```diff
@@ -178,7 +179,9 @@ public void execute() throws Exception {
                 .generateSpring(workParameter.generateSpring)
                 .metricsCollector(metricsCollector)
                 .generateOptionalTypes(workParameter.generateOptionalTypes)
-                .generatePayloadValidator(workParameter.generatePayloadValidator));
+                .generatePayloadValidator(workParameter.generatePayloadValidator)
+                .useCache(workParameter.useCache)
+                .useCache(workParameter.useCache));
```

Comment:
I'm pretty sure `useCache` will be set to `workParameter.useCache`'s value this way:D